jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalTopicDraftService.java

```diff
@@ -61,8 +62,10 @@ public TopicDraft saveOrUpdateDraft(TopicDraft draft) {
         } else {
             currentDraft.setContent(draft.getContent());
             currentDraft.setTitle(draft.getTitle());
-            currentDraft.setPollTitle(draft.getPollTitle());
-            currentDraft.setPollItemsValue(draft.getPollItemsValue());
+            if (draft.getPollTitle() != null || draft.getPollItemsValue() != null) {
```

Comment:
We need this condition because when we save draft that does not contain poll, we should not overwrite already exists values. And now, when user removes the text from field, instead of sending null, we will send empty string.

I have added comment to this condition in the next commit.